src/_pytest/debugging.py

```diff
@@ -98,6 +99,11 @@ def _import_pdb_cls(cls):
         if not pdb_cls:
             return pdb.Pdb
 
+        if cls._wrapped_pdb_cls and cls._wrapped_pdb_cls[0] == pdb_cls:
+            return cls._wrapped_pdb_cls[1]
+
+        import _pytest.config
```

Comment:
This is used only for `tw = _pytest.config.create_terminal_writer(cls._config)` below.